docs/source/api_reference/layers.rst

```diff
@@ -248,3 +248,6 @@ Density Functional Theory Layers
 
 .. autoclass:: deepchem.models.dft.scf.XCNNSCF
    :members:
+
+.. autoclass:: deepchem.models.dft.dftxc.DFTXC
```

Comment:
Can you add the DFT layers to the new table

Reply:
Documenting offline discussion, to be done in a follow up PR, with DFT layers to be added to https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/torch_layers.csv